src/main/resources/org/csanchez/jenkins/plugins/kubernetes/PodTemplate/config.jelly

```diff
@@ -93,12 +93,11 @@
     <f:textbox/>
   </f:entry>
 
-    <f:optionalBlock title="${%Use custom workspace volume}" field="customWorkspaceVolumeEnabled" inline="true">
-      <f:entry title="${%Workspace Volume}" description="${%Volume to use for sharing the workspace}">
-        <f:dropdownDescriptorSelector field="workspaceVolume" oneEach="true" hasHeader="true" addCaption="${%Add Workspace Volume}"
-                                      deleteCaption="${%Delete Workspace Volume}" />
-      </f:entry>
-    </f:optionalBlock>
+  <f:entry title="${%Workspace Volume}" description="${%Volume to use for sharing the workspace}">
+    <f:dropdownDescriptorSelector field="workspaceVolume" oneEach="true" hasHeader="true" addCaption="${%Add Workspace Volume}"
+                                  default="${descriptor.defaultWorkspaceVolume}"
+                                  deleteCaption="${%Delete Workspace Volume}" />
```

Comment:
```suggestion
    <f:dropdownDescriptorSelector field="workspaceVolume" default="${descriptor.defaultWorkspaceVolume}"/>
```

since these other attrs AFAICT are only for hetero lists, not dropdowns.